sherpa/optmethods/__init__.py

```diff
@@ -372,8 +372,8 @@ class LevMar(OptMethod):
        free parameters.
     epsfcn : number
        This is used in determining a suitable step length for the
-       forward-difference approximation; default is sqrt( DBL_EPSILON
-       ) ~ 1.19209289551e-07, where DBL_EPSILON is the smallest number
+       forward-difference approximation; default is FLT_EPSILON ~ 1.19209289551e-07,
+       where DBL_EPSILON is the smallest number
```

Comment:
should this be DBL_EPSILON or FLT_EPSILON? I'm guessing replace DBL_EPSILON by FLT_EPSILON as done elsewhere (e.g. the changes for `gtol`).

I have tried hard to keep the docstring lines "short", so they can be read easily in an 80-column
terminal. Can line 375 be broken up at the ~ (e.g. as done in line 364)?